server/test/api/patients.js

```diff
@@ -56,8 +56,7 @@ describe('The /patients API', function () {
   var mockMissingRequest = {
     finance : {
       debitor_group_uuid : '4de0fe47-177f-4d30-b95f-cff8166400b4'
-    },
-    medical : missingParamsPatient
```

Comment:
@jniles I changed because all the walking test integration with the exeption of " POST / patients will reject a patient with missing information"